CANTroller2/platformio.ini

```diff
@@ -13,6 +13,8 @@ default_envs = esp32-s3-devkitc-1
 
 [env]
 framework = arduino
+; baud rates: 9600/19200/28800/57600/115200/230400/460800/921600
+custom_baudrate = 921600
```

Comment:
We now can just set this in one place and it is used in the 3 places below. I tested it locally.